### PR TITLE
waybar: add page

### PR DESCRIPTION
**Pull request by TechnoFlask**
Waybar is a highly customizable Wayland bar for Sway and Wlroots based compositors.

- [x] The page(s) are in the correct platform directories: `common`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/waybar.md
Comment (spageektti) [inline pages/common/waybar.md:16]:
```suggestion
`waybar {{-s|--style}} {{path/to/stylesheet.css}}`
```
The stylesheet is in CSS format according to the official wiki.

File: pages/common/waybar.md
Comment (spageektti) [inline pages/common/waybar.md:12]:
```suggestion
`waybar {{-c|--config}} {{path/to/config.jsonc}}`
```
The config file is in JSONC format according to the official wiki.

**Review by spageektti — APPROVED** [review]
Hi. Thanks for your contribution. I left some suggestions.

**Review by spageektti — APPROVED** [review]
LGTM.

**Comment by TechnoFlask** [comment]
What should I do now? I need 2 reviewers to merge this pr. But I was only allocated 1. Is there anything I need to do to complete this procedure

**Comment by spageektti** [comment]
> What should I do now? I need 2 reviewers to merge this pr. But I was only allocated 1. Is there anything I need to do to complete this procedure

You don't have to do anything. We have to wait for another maintainer to review.

**Comment by TechnoFlask** [comment]
Oh ok

**Review by acuteenvy — APPROVED** [review]
> I need 2 reviewers to merge this pr. But I was only allocated 1.

All maintainers can review PRs. The "allocations" you're seeing are automatic review requests sent to maintainers that have put themselves in the [`CODEOWNERS`](https://github.com/tldr-pages/tldr/blob/main/.github/CODEOWNERS) file, but that doesn't mean anything - it just sends a notification to the maintainer.

**Pull request merged**
